app/models/license.rb

```diff
@@ -1,13 +1,16 @@
-# typed: strict
+# typed: true
 # frozen_string_literal: true
 
-# Represents the list of valid work types
+# Class contains valid license types, their identifiers from https://spdx.org/licenses/
+#  and other structures used in the work form and its validation
 class License
   extend T::Sig
 
+  # identifier from https://spdx.org/licenses/
   sig { returns(String) }
   attr_reader :id
```

Comment:
Since you removed all the `new()` calls, I think you can remove the attributes too.